www/data_module/src/mock/generator.service.coffee

```diff
@@ -0,0 +1,31 @@
+# types for generating test data: null, number, string, boolean, timestamp, <array>[], <object>, <objectName in Specification>
+class Generator extends Service
+    self = null
+    constructor: ->
+        self = @
+
+    number: (min = 0, max = 100) ->
+        random = Math.random() * (max - min) + min
```

Comment:
Its usually not very cool to use random for generating faked data.

It means that your data is not reproducible